src/components/checkbox/index.tsx

```diff
@@ -177,4 +209,4 @@ function createStyles({color = DEFAULT_COLOR, iconColor = DEFAULT_ICON_COLOR, si
   });
 }
 
-export default Checkbox;
+export default asBaseComponent<CheckboxProps>(forwardRef(Checkbox));
```

Comment:
and this...In general, the whole use of the offered  HOCs from `'../../commons/new'` is not crystal clear to me when it comes to knowing what to use and when :)